lib/engine_http.js

```diff
@@ -172,6 +174,20 @@ HttpEngine.prototype.step = function step(requestSpec, ee, opts) {
         {});
     }
 
+    if (params.formData) {
+      requestParams.formData = _.reduce(
+        requestParams.formData,
+        function (acc, v, k) {
+          const reference = fileReference(v, config);
```

Comment:
Aren't you passing formData with read stream?
Like this:
            formData: {
                foo: {
                    value: fs.createReadStream(filePath),
                    options: {
                        filename: fileName,
                        contentType: 'application/octet-stream'
                    }
                }
            }